### PR TITLE
Cantata new recipe

### PR DESCRIPTION
**Pull request by miqlas**
a26607259b8347419eaedd30f624493b

### CODE REVIEW

File: media-sound/cantata/cantata-2.0.1.recipe [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+export Qt5Core_DIR="/boot/system/data/cmake/Modules/Qt5Core"
+export Qt5DBus_DIR="/boot/system/data/cmake/Modules/Qt5DBus"
+export Qt5LinguistTools_DIR="/boot/system/data/cmake/Modules/Qt5LinguistTools"
+export Qt5Multimedia_DIR="/boot/system/data/cmake/Modules/Qt5Multimedia"
+export Qt5Network_DIR="/boot/system/data/cmake/Modules/Qt5Network"
+export Qt5Test_DIR="/boot/system/data/cmake/Modules/Qt5Test"
+export Qt5Sql_DIR="/boot/system/data/cmake/Modules/Qt5Sql"
+export Qt5Svg_DIR="/boot/system/data/cmake/Modules/Qt5Svg"
+export Qt5Xml_DIR="/boot/system/data/cmake/Modules/Qt5Xml"
+export Qt5Widgets_DIR="/boot/system/data/cmake/Modules/Qt5Widgets"
```
Comment (miqlas) [inline media-sound/cantata/cantata-2.0.1.recipe]: True, but i can't compile big things like Qt on my small laptop, so i need to use them from HaikuDepot. Maybe you can update the packages.

Comment (korli) [inline media-sound/cantata/cantata-2.0.1.recipe]: hi miqlas, Qt is up to date on x86_64. Have fun!

Comment (miqlas) [inline media-sound/cantata/cantata-2.0.1.recipe]: Korli, i'm using gcc2h for my ports, x64 Haiku installed just for testing. I just built the latest cmake from haikuports, but it stil lrequires theese exports. Should i try to rebuild Qt too for gcc2h?

**Comment by korli** [comment]
I tested that it launches correctly on x86_64, but not much more.

**Pull request merged**